src/index.js

```diff
@@ -119,5 +162,36 @@ export default ({
   // for locally served modules
   oneAppDevCdn.use('/modules', express.static(`${localDevPublicPath}/modules`));
 
+  // eslint-disable-next-line consistent-return
+  oneAppDevCdn.get('*', (req, res) => {
```

Comment:
I know this is ugly but cannot figure out a different way of doing it, definitely open to feedback!